### DISCUSSION TITLE
Setting `lane_markings=no` disables setting `lanes`

### DISCUSSION THREAD

**Discussion by habi** [streetcomplete/StreetComplete#4982] [Feedback]
While discussing a little 'data gardening' issue in Switzerland [1] it was noted that when StreetComplete asks for the number of traffic lanes and one adds "unpainted", then you don't have the possibility to enter the number of lanes of the street.

In Switzerland there are often streets with 2 lanes that have no lane marking, i.e. that have clearly 2 lanes, but are not painted, and it seems to me more interesting in these cases to add `lanes=2` than the absence of paint.

I first wanted to discuss this here before potentially opening an issue.
What are your thoughts about this?

(I'm on iOS, so not a SC user)

[1]: http://lists.openstreetmap.ch/pipermail/talk-ch/2023-April/011897.html and following threads

**Comment by HolgerJeromin** [comment]
ref [#2553](https://github.com/streetcomplete/StreetComplete/issues/2553#issuecomment-773638420)
and #2299 for problems with that.

**Reply by Helium314** [reply]
and also #2774

**Comment by mnalis** [comment]
> StreetComplete asks for the number of traffic lanes and one adds "unpainted", then you don't have the possibility to enter the number of lanes of the street.

As note by others, that is not asked because it has problems. But, StreetComplete will ask you (if you have that quest enabled) to enter [verifiable](https://wiki.openstreetmap.org/wiki/Verifiability) information, like [width](https://wiki.openstreetmap.org/wiki/Key:width) of the street (and/or if it is [oneway](https://wiki.openstreetmap.org/wiki/Key:oneway) street)
